packages/web/src/pages/rewards-page/components/modals/ChallengeRewardsModal/DefaultChallengeContent.tsx

```diff
@@ -0,0 +1,185 @@
+    challenge &&
+    challenge.max_steps &&
+    challenge.max_steps > 1 &&
+    challenge.challenge_type !== 'aggregate'
+
+  const progressDescription = (
+    <Flex column gap='m' w='100%'>
+      {isVerifiedChallenge ? (
+        <div className='verifiedChallenge'>
+          <IconVerified />
+          {messages.verifiedChallenge}
+        </div>
+      ) : null}
+      <Text variant='body'>{fullDescription?.(challenge)}</Text>
+      {challenge?.cooldown_days && challenge.cooldown_days > 0 ? (
+        <Text variant='body' color='subdued'>
+          {messages.cooldownDescription}
+        </Text>
+      ) : null}
+    </Flex>
+  )
+
+  const progressStatusLabel = (
+    <Flex
+      alignItems='center'
+      border='strong'
+      w='100%'
+      justifyContent='center'
+      ph='xl'
+      pv='unit5'
+      borderRadius='s'
+      backgroundColor='surface1'
+    >
+      <Flex alignItems='center' gap='s'>
+        {challenge?.state === 'completed' ||
+        challenge?.state === 'disbursed' ||
+        (challenge?.claimableAmount ?? 0) > 0 ? (
+          <IconCheck size='s' color='subdued' />
+        ) : null}
+        <Text variant='label' size='l' strength='strong' color='subdued'>
+          {progressLabel
+            ? fillString(
+                progressLabel,
+                formatNumberCommas(
+                  challenge?.current_step_count?.toString() ?? ''
+                ),
+                formatNumberCommas(challenge?.max_steps?.toString() ?? '')
+              )
+            : null}
+        </Text>
+      </Flex>
+    </Flex>
+  )
+
+  const goToRoute = useCallback(
+    (route: string) => {
+      dispatch(pushRoute(route))
+      onNavigateAway()
+    },
+    [dispatch, onNavigateAway]
+  )
+
+  const buttonLink = (challenge as any)?.buttonLink
+  const buttonLabel = (challenge as any)?.buttonLabel
+
+  const renderAdditionalContent = () => {
+    const contents: ReactNode[] = []
+
+    if (challengeName === ChallengeName.ProfileCompletion) {
+      contents.push(<ProfileChecks key='profile-checks' />)
+    }
+    if (
+      challengeName === ChallengeName.MobileInstall ||
+      challengeName === ChallengeName.ConnectVerified
+    ) {
+      contents.push(<MobileInstallContent key='mobile-install' />)
+    }
+    if (challenge?.cooldown_days && challenge.cooldown_days > 0) {
+      contents.push(
+        <CooldownSummaryTable
+          key='cooldown-summary'
+          challengeId={challenge.challenge_id}
+        />
+      )
+    }
+
+    return contents.length > 0 ? (
+      <Flex w='100%' column gap='2xl'>
+        {contents}
+      </Flex>
+    ) : null
+  }
+
+  return (
+    <ChallengeRewardsLayout
+      description={progressDescription}
+      amount={challenge?.totalAmount ?? undefined}
+      rewardSubtext={messages.audio}
+      progressStatusLabel={progressStatusLabel}
+      progressValue={
+        showProgressBar
+          ? (challenge?.current_step_count ?? undefined)
+          : undefined
```

Comment:
im a bit confused at the `?? undefined` here

Reply:
agreed, i guess it's dont show if 0? but removed for now